src/UI/class-settings-page.php

```diff
@@ -108,250 +121,274 @@ public function initialize_settings(): void {
-				'help_text'        => $h,
-				'requires_recrawl' => false,
+				'title'      => __( 'Track Logged-in Users', 'wp-parsely' ), // Passed for legend element.
+				'option_key' => 'track_authenticated_users',
+				'help_text'  => $h,
 			)
 		);
 
-		/* translators: 1: Opening anchor tag markup, 2: Documentation URL, 3: Opening anchor tag markup continued, 4: Closing anchor tag */
-		$h          = __( 'Choose the metadata format for our crawlers to access. Most publishers are fine with JSON-LD ( %1$s%2$s%3$shttps://www.parse.ly/help/integration/jsonld/%4$s ), but if you prefer to use our proprietary metadata format then you can do so here.', 'wp-parsely' );
-		$h_link     = 'https://www.parse.ly/help/integration/jsonld/';
-		$field_id   = 'meta_type';
+		// Disable JavaScript.
+		$h = __( 'If you use a separate system for JavaScript tracking (Tealium / Segment / Google Tag Manager / other tag manager solution) you may want to use that instead of having the plugin load the tracker. WARNING: disabling this option will also disable the "Personalize Results" section of the recommended widget! We highly recommend leaving this option set to "No".', 'wp-parsely' );
+		add_settings_field(
+			'disable_javascript',
+			__( 'Disable JavaScript', 'wp-parsely' ),
+			array( $this, 'print_binary_radio_tag' ),
+			Parsely::MENU_SLUG,
+			'basic_settings',
+			array(
+				'title'      => __( 'Disable JavaScript', 'wp-parsely' ), // Passed for legend element.
+				'option_key' => 'disable_javascript',
+				'help_text'  => $h,
+			)
+		);
+
+		if ( defined( 'AMP__VERSION' ) ) {
+			// Disable AMP tracking.
+			$h = __( 'If you use a separate system for JavaScript tracking on AMP pages (Tealium / Segment / Google Tag Manager / other tag manager solution) you may want to use that instead of having the plugin load the tracker.', 'wp-parsely' );
+			add_settings_field(
+				'disable_amp',
+				__( 'Disable AMP Tracking', 'wp-parsely' ),
+				array( $this, 'print_binary_radio_tag' ),
+				Parsely::MENU_SLUG,
+				'basic_settings',
+				array(
+					'title'      => __( 'Disable AMP Tracking', 'wp-parsely' ), // Passed for legend element.
+					'option_key' => 'disable_amp',
+					'help_text'  => $h,
+				)
+			);
+		}
+
+		// These are the Requires Recrawl Settings.
+		add_settings_section(
+			'requires_recrawl_settings',
+			__( 'Requires Recrawl Settings', 'wp-parsely' ),
+			function (): void {
+				echo '<strong>' . wp_kses_post( __( '<span style="color:#d63638">Important:</span> changing any of these values below on a site currently tracked with Parse.ly will require reprocessing of your Parse.ly data.', 'wp-parsely' ) ) . '</strong><br />';
+				printf(
+					/* translators: Mailto link  */
+					esc_html__( 'Once you have changed a value and and saved, please contact %s to request a recrawl.', 'wp-parsely' ),
+					wp_kses_post( '<a href="mailto:support@parsely.com?subject=Please reprocess ' . $this->parsely->get_api_key() . '">support@parsely.com</a>' )
```

Comment:
May I suggest `urlencode()`'ing the subject? I remember I once had an issue with an iPhone and an unencoded whitespace character in the link.

Reply:
Thanks - I've handled this in #518, and then rebased this PR onto the updated #518.